tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -278,6 +278,38 @@ def test_flatnonzero(self, xp, dtype):
         return xp.flatnonzero(array)
 
 
+@testing.parameterize(
+    {'array': numpy.random.randint(0, 2, (20,))},
+    {'array': numpy.random.randn(3, 2, 4)},
+    {'array': numpy.empty((0,))},
+    {'array': numpy.empty((0, 2))},
+    {'array': numpy.empty((0, 2, 0))},
+)
+@testing.gpu
+class TestArgwhere(unittest.TestCase):
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal()
+    def test_argwhere(self, xp, dtype):
+        array = xp.array(self.array, dtype=dtype)
+        return xp.argwhere(array)
+
+
+@testing.parameterize(
+    {'array': numpy.array(0)},
+    {'array': numpy.array(1)},
+)
+@testing.gpu
+@testing.with_requires('numpy>=1.17.0')
+class TestArgwhereZeroDimension(unittest.TestCase):
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_raises(accept_error=DeprecationWarning)
```

Comment:
I don't think this is OK. Warnings are not exceptions. Please change this line and line 256 (yeah it was not correctly done in the codebase) and use the context manager `with testing.assert_warns(DeprecationWarning)` instead.

Reply:
Done